models/componentMessage.go

```diff
@@ -68,9 +68,26 @@ func (ths *MessageComponent) Validate() (bool, []error) {
 }
 
 // NewMessage returns a new instance of a message component to be rendered
-func NewMessage() *MessageComponent {
+func NewMessage(tenantID, brandID, roomID, channel string) *MessageComponent {
 	var c MessageComponent
 	c.Type = MCTMessage
+	c.Message.TenantID = tenantID
+	c.Message.BrandID = brandID
+	c.Message.RoomID = roomID
+	c.Message.TenantID = tenantID
 	c.component.IComponent = &c
 	return &c
 }
+
+func NewMessageTextObject(body string) *MessageTextObject {
```

Comment:
ini harusnya array ya mas untuk texts nya

Reply:
ku pikir kayaknya lebih fleksibel bagi creator untuk tetep bisa instantiate MessageTextObject sebagai bentuk satuan. tapi sepertinya perlu ada juga generator buat yang langsung arraynya. akan ku tambahin `NewMessageTextObjects()` atau `NewMessageTextObjectArray()`

Reply:
ga perlu mas kalo functionnya kek gt oke sih, tapi prosesnya lgsg diappend aja kek yg di carousel
ini contohnya
`func (s *CarouselBlock) AddImageCarousel(alt string, src string) {
	image := ImageBlock{
		Image: &ImageBlockObject{
			Src: src,
			Alt: alt,
		},
	}
	image.Type = MBTImage
	s.Carousel.Images = append(s.Carousel.Images, image)
}`

Reply:
jadi nanti ngecallnya bisa di for dengan call function nya tapi di proses nya tuh dia udah append2 trus, mirip banget sama di carousel implementnya, jadi bisa liat disitu

Reply:
I see. oke ku update dulu

Reply:
mungkin untuk append2nya aku samain persis kayak yang di carousel. tapi function untuk new message text blocknya tetep dipertahankan. gimana mas kalo begitu?

Reply:
lebih tepatnya function NewMessage() yg di pertahankan, trus tinggal buat function buat append texts sama images